src/pal/src/map/map.cpp

```diff
@@ -2677,3 +2677,54 @@ BOOL MAPUnmapPEFile(LPCVOID lpAddress)
     TRACE_(LOADER)("MAPUnmapPEFile returning %d\n", retval);
     return retval;
 }
+
+/*++
+Function :
+    MAPEraseSection - erase a section
+    returns TRUE if successful, FALSE otherwise
+--*/
+BOOL MAPEraseSection(LPCVOID lpAddress)
```

Comment:
Can you please update the function name / comments since we are not erasing the section anymore? E.g. `MAPMarkSectionAsNotNeeded` or something along these lines.

Reply:
Thank you. I updated the name of functions.